contracts/tokenomics/maker/src/contract.rs

```diff
@@ -67,6 +79,35 @@ pub fn instantiate(
     Ok(Response::default())
 }
 
+/// ## Description
+/// Available the execute messages of the contract.
+/// ## Params
+/// * **deps** is the object of type [`Deps`].
+///
+/// * **env** is the object of type [`Env`].
+///
+/// * **_info** is the object of type [`MessageInfo`].
+///
+/// * **msg** is the object of type [`ExecuteMsg`].
+///
+/// ## Queries
+/// * **ExecuteMsg::Collect { pair_addresses }** Collects astro tokens. Before that,
```

Comment:
Collects rewards from the pools, swaps to astro token and distributes the rewards between staking and governance contracts